alphagov/paas-cf
# [#155824421] Always deploy billing app

## What

We came to the conclusion, there is no real reason to restrict deploying
this application to all of our environments. It takes more time to
deploy that application when it is actually needed than if it would be
done for us every morning.

We also thought, that leaving ability to disable deployment of the
package, clearly indicates problems with our deployment and should be
targeted when it becomes an issue instead.

Now that the billing app will be deployed to all environments at all
times, we could save some money by restricting the service plan it
should use for different environments.

We went with the smallest available plan "Free" for dev and staging,
which should be updated once we rename the naming as we don't think it
should require more resources for small environments.

## How to review

- Code sanity check
- Deploy from this branch
- Run `create-cloudfoundry` pipeline
- See if `post-deploy` fully executed `deploy-paas-billing` task
